lib/streamDAO.js

```diff
@@ -270,6 +273,9 @@ module.exports = function(mongoClient){
       pre.hasProperty(datum, 'id');
       pre.hasProperty(datum, '_userId');
       pre.hasProperty(datum, '_groupId');
+      if (!datum.modifiedTime) {
```

Comment:
Same as in Insert, we don't want any client tampered modifiedTime from clients, this should be set unconditionally.